auction-cordapp/workflows/src/test/java/net/corda/samples/FlowTests.java

```diff
@@ -0,0 +1,74 @@
+package net.corda.samples;
+
+import com.google.common.collect.ImmutableList;
+import net.corda.core.concurrent.CordaFuture;
+import net.corda.core.contracts.Amount;
+import net.corda.core.transactions.SignedTransaction;
+import net.corda.samples.flows.BidFlow;
+import net.corda.samples.flows.CreateAssetFlow;
+import net.corda.samples.flows.CreateAuctionFlow;
+import net.corda.samples.states.Asset;
+import net.corda.samples.states.AuctionState;
+import net.corda.testing.node.MockNetwork;
+import net.corda.testing.node.MockNetworkParameters;
+import net.corda.testing.node.StartedMockNode;
+import net.corda.testing.node.TestCordapp;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+
+import java.time.Instant;
+import java.time.LocalDateTime;
+import java.time.ZoneId;
+import java.util.UUID;
+
+import static org.junit.Assert.*;
+
+public class FlowTests {
+    private  MockNetwork network;
+    private  StartedMockNode a;
+    private  StartedMockNode b;
+
+    @Before
+    public void setup() {
+        network = new MockNetwork(
+                new MockNetworkParameters(
+                        ImmutableList.of(
+                            TestCordapp.findCordapp("net.corda.samples.flows"),
+                            TestCordapp.findCordapp("net.corda.samples.contracts")
+                        )
+                )
+        );
+        a = network.createPartyNode(null);
+        b = network.createPartyNode(null);
+        network.runNetwork();
+    }
+
+    @After
+    public void tearDown() {
+        network.stopNodes();
+    }
+
+    @Test
+    public void testCreateAssetFlow() throws Exception {
+        CreateAssetFlow flow = new CreateAssetFlow("Test Asset", "Dummy Asser", "http://abc.com/dummy.png");
+        CordaFuture<SignedTransaction> future = a.startFlow(flow);
+        SignedTransaction signedTransaction = future.get();
+        Asset asset = (Asset) signedTransaction.getTx().getOutput(0);
+        assertNotNull(asset);
+    }
+
+    @Test
+    public void testCreateAuctionFlow() throws Exception {
```

Comment:
Second test fails on my system.
```
net.corda.core.node.ZoneVersionTooLowException: Reference states requires all nodes on the Corda compatibility zone to be running at least platform version 4. The current zone is only enforcing a minimum platform version of 1. Please contact your zone operator.
```

Likely some setting with MockNetwork. I didn't see it off the bat though. Will look again later. But if it isn't easy to correct this we should eliminate this test as users will be confused.

Reply:
Yah, figued this out, updated the mPV to 4